pkgs/development/python-modules/nipy/default.nix

```diff
@@ -37,6 +37,10 @@ buildPythonPackage rec {
 
   postPatch = ''
     patchShebangs nipy/_build_utils/cythoner.py
+
+    substituteInPlace lib/fff_python_wrapper/fffpy.c \
```

Comment:
Please consider linking to upstream issue via a comment, not just in the PR title.